extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -61,6 +61,37 @@ static public function newFromId( $id, $master = false ) {
 		return null;
 	}
 
+	/**
+	 * @param array $ids
+	 * @return array
+	 */
+	static public function newFromIds( $ids ) {
+		wfProfileIn( __METHOD__ );
+
+		$titles = Title::newFromIDs( $ids );
+		$wallMessages = [ ];
+		$correctIds = [ ];
+
+		//double check if all titles are correct
+		foreach ( $titles as $title ) {
+			if ( $title instanceof Title && $title->exists() ) {
```

Comment:
Calling `Title::newFromIds()` we expect `$titles` to be an array of `Title`s. Therefore the `instanceof` check is not needed or we should log cases when an item is not a `Title`. I would even go with assertions but it might be too aggressive at the moment. So please, log non-Titles as errors with label `SUS-165` to Kibana.